content/applications/sales/point_of_sale.rst

```diff
@@ -48,28 +47,17 @@ Once an order is completed, proceed to checkout by clicking :guilabel:`Payment`.
 
 .. _pos/customers:
 
-Register customers
-==================
-
-Registering your customers gives you the ability to grant them various privileges such as discounts,
-loyalty program, specific communication. It is also required if they want an invoice and registering
-them makes any future interaction with them faster.
-
-From your session interface, use the customer button.
-
-.. image:: point_of_sale/customer-button.png
-
-Create a new one by using this button.
-
-.. image:: point_of_sale/customer-add.png
-
-You will be invited to fill out the customer form with their
-information.
+Set customers
+=============
 
-.. image:: point_of_sale/customer-form.png
+You can create and set customers from an :ref:`open POS session <pos/session-start>`. Registering
+your customer is necessary to :doc:`collect their loyalty points and grant them rewards
+<point_of_sale/pricing/loyalty>`, automatically apply the :doc:`attributed pricelist
+<point_of_sale/pricing/pricelists>`, or :ref:`generate and print an invoice <receipts/invoices>`.
 
-Use the save button when you are done. You can then select that customer
-in any future transactions.
+To access the list of customers, click :guilabel:`Customer` on the POS interface. Then, select a
+customer or create a new one by clicking :guilabel:`Create`, completing the form, and saving.
+Lastly, click :guilabel:`Set Customer` to set the selected customer or your newly created client.
```

Comment:
```suggestion
Lastly, click :guilabel:`Set Customer` to set the selected customer.
```
If feel like the "newly created client" is included in the "selected customer", since they're selected in bioth cases? (if you still want to keep "newly created client", I would write "the newly created ..." instead of "your ..."